src/main/kotlin/org/rust/ide/console/RsConsoleCodeFragmentContext.kt

```diff
@@ -57,15 +57,21 @@ class RsConsoleCodeFragmentContext {
 }
 
 class RsConsoleOneCommandContext(codeFragment: RsReplCodeFragment) {
-    val topLevelElements: MutableMap<String, String> = mutableMapOf()
+    val topLevelItems: MutableMap<String, String> = mutableMapOf()
```

Comment:
Do we really want to split all elements into "items" and "statements"?
Probably it may lead to bugs because order matters:
```Rust
use foo;
stmt_1;
use bar;
stmt_2;
```
can be semantically different from
```Rust
use foo;
use bar;
stmt_1;
stmt_2;
```
But maybe I'm wrong and it is OK, what do you think?

Reply:
Originally we split elements into items and statements, because we want to do something with items which have same name (they are not allowed in rust functions, but are allowed in evcxr repl). This approach can indeed lead to some bugs (please see new tests which check redefining function/struct). I remove that splitting and now we use nested scopes for next commands, like so:

```rust
fn main() {
    command1;
    {
        command2;
        {
            command3;
        }
    }
}
```
And `RsBlock` surrounding last command (`command3`) will become context of codeFragment